content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md

```diff
@@ -129,14 +129,7 @@ Make sure that in the current branch:
   php bin/console prestashop:licenses:update
   ```
   
-* All controllers are secured by annotations, and legacy link are provided for Symfony routes:
-  
-  ```bash
-  php bin/console prestashop:linter:security-annotation
-  php bin/console prestashop:linter:legacy-link
-  ```
-  
-* There are no known vulnerabilities in composer dependencies using [Fabpot Local PHP Security Checker][security-checker]. Consider using [this][security-checker-installer] if installing Fabpot Security Checker proves troublesome.
+* Please execute `composer audit` to check vulnerabilities.
```

Comment:
Ok for the replacement for security
However, we should let the steps for version 8.* and indicate that they are no longer need for versions 9+